File: backend-dotnet/SahayaSetu/DTO/RequestResponseDto.cs
using SahayaSetu.Models;

namespace SahayaSetu.DTO
{
    public class RequestResponseDto
    {
        public int RequestId { get; set; }
        public int NgoId { get; set; }
        public NgoDto? Ngo { get; set; }
        public string? Title { get; set; }
        public string? Description { get; set; }
        public string? Status { get; set; }
        public string? RequestType { get; set; }
        public DateTime CreatedAt { get; set; }
    }

    public class NgoDto
    {
        public string? Name { get; set; }
    }
}

File: backend-java/sahayaSetu/src/main/java/com/sahayaSetu/dtos/RequestResponseDto.java
package com.sahayaSetu.dtos;

import java.time.LocalDateTime;

import com.sahayaSetu.entities.enums.RequestStatus;
import com.sahayaSetu.entities.enums.RequestType;

import lombok.Data;

@Data
public class RequestResponseDto {
    private Long requestId;
    private String title;
    private String description;
    private RequestType requestType;
    private RequestStatus status;
    private LocalDateTime createdAt;
    private NgoResponseDto ngo;
}
